db/migrate/20221005190528_remove_old_race_gender.rb

```diff
@@ -0,0 +1,10 @@
+class RemoveOldRaceGender < ActiveRecord::Migration[6.0]
+  def change
+    remove_column :project_clients, :primary_race, :string
+    remove_column :project_clients, :secondary_race, :string
+    remove_column :project_clients, :gender, :integer
+    remove_column :clients, :race_id, :integer
+    remove_column :clients, :gender_id, :integer
+    remove_column :clients, :gender_other, :string
```

Comment:
I think we should leave these temporarily.  If we remove them the two deployments need to go in lock-step or the CAS sync process will start bombing out as soon as we've deployed one or the other.  If we leave them, deploy both, then, remove them, that should maintain the dependency.